mod/metrics/spec/lib/formula/javascript/script_metrics.test.coffee.off

```diff
@@ -0,0 +1,21 @@
+CARD_MOD_DIR = "../../../../../../vendor/decko/card/mod/";
+DECKO_JS_DIR = CARD_MOD_DIR + "/machines/lib/javascript/";
+
+# commenting for now, because export is not working outside of Jest
+# (presumably because it's a Nod
```

Comment:
note this comment, even though I failed to finish the thought.  I think the export thing is a node thing.